### PR TITLE
Fix code formatting

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: app/models/spree/calculator/shipping/active_shipping/base.rb [hunk 1]
```diff
@@ -254,11 +254,11 @@ def convert_package_to_dimensions_array(package)
 
         # Generates an array of Package objects based on the quantities and weights of the variants in the line items
         def packages(package)
-          units = Spree::ActiveShipping::Config[:units].to_sym
-          packages = []
-          weights = convert_package_to_weights_array(package)
-          max_weight = get_max_weight(package)
-          dimensions = convert_package_to_dimensions_array(package)
+          units                  = Spree::ActiveShipping::Config[:units].to_sym
```
Comment (mbj) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: can we use `#fetch` here? Or does `Config` not support it?

**Comment by mbj** [comment]
@dkubb I finished a code only pass

File: app/models/spree/calculator/shipping/active_shipping/base.rb (hunk 1 above)
Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: `Spree::ActiveShipping::Config` returns a `Spree::ActiveShippingConfiguration` instance, which does not respond to `#fetch` unfortunately.

**Comment by dkubb** [comment]
@mbj this is ready for another pass.

**Pull request merged**